server.go

```diff
@@ -2924,22 +2924,6 @@ func extractChannelUpdate(info *channeldb.ChannelEdgeInfo,
 // applyChannelUpdate applies the channel update to the different sub-systems of
 // the server.
 func (s *server) applyChannelUpdate(update *lnwire.ChannelUpdate) error {
-	newChannelPolicy := &channeldb.ChannelEdgePolicy{
```

Comment:
Nice catch!

Reply:
Heh was meaning to remove this myself once I realized it wasn't needed 👍